src/normalize/entities.ts

```diff
@@ -311,6 +311,10 @@ const FreeBusyInstance = new Entity({
 	s: 'start',
 	e: 'end'
 });
+const WorkingHourInstance = new Entity({
```

Comment:
looks like same as FreeBusyInstance

Reply:
removed this as we do not need new entity